app/smc/sysbuild.cmake

```diff
@@ -62,86 +71,70 @@ if(NOT "${BOARD_REVISION}" STREQUAL "galaxy")
   )
 endif()
 
-if (SB_CONFIG_BL2)
-  # Make sure MCUBoot is build only
-  set_target_properties(mcuboot PROPERTIES BUILD_ONLY 1)
+# Make sure MCUBoot is build only
+set_target_properties(mcuboot PROPERTIES BUILD_ONLY 1)
 
-  # This is quite a hack- in order to get the mcuboot hook file we need to override
-  # image verification (which allows us to skip signature checks) into the mcuboot
-  # build, we need to create a tiny Zephyr module in the source directory, and
-  # tell sysbuild to include that module with mcuboot
-  set(mcuboot_EXTRA_ZEPHYR_MODULES "${CMAKE_CURRENT_LIST_DIR}/mcuboot_module" CACHE INTERNAL "mcuboot_module directory")
-endif()
+# This is quite a hack- in order to get the mcuboot hook file we need to override
+# image verification (which allows us to skip signature checks) into the mcuboot
+# build, we need to create a tiny Zephyr module in the source directory, and
+# tell sysbuild to include that module with mcuboot
+set(mcuboot_EXTRA_ZEPHYR_MODULES "${CMAKE_CURRENT_LIST_DIR}/mcuboot_module" CACHE INTERNAL "mcuboot_module directory")
+set_config_string(mcuboot CONFIG_BOOT_SIGNATURE_KEY_FILE "${SB_CONFIG_BOOT_SIGNATURE_KEY_FILE}")
```

Comment:
Should we be checking if `SB_CONFIG_BOOT_SIGNATURE_KEY_FILE` is empty here? Or will it get a default value of the key in the mcuboot repo?

Reply:
Perfect- if you have the bandwidth, let's drop the other checks in this file then?